invenio_records_marc21/ui/theme/assets/semantic-ui/js/invenio_records_marc21/deposit/fields/MetadataFields.js

```diff
@@ -61,6 +61,10 @@ export class MetadataFields extends Field {
 
   static _deserialize_subfields(subfields) {
     let field = ""; // subfields.join(" ");
+    // Empty datafield leave subfield in UI empty
+    if (subfields == null) {
```

Comment:
How is it possible that subfields is null?

Reply:
If the user leaves the Textfield for the subfield empty it might be possible the datafield in the record has no subfield present and the react application crashes at rendering.